### PR TITLE
BP-325 add sora plugin

### PR DESCRIPTION
**Pull request by bbr88**

### CODE REVIEW

File: settings.gradle [hunk 1]
```diff
@@ -13,4 +13,4 @@ pluginManagement {
     }
 }
 
-rootProject.name = "local-map-service"
+rootProject.name = "bakong-local-map-service"
```
Comment (BulatSaif) [inline settings.gradle:16]:
```suggestion
rootProject.name = "local-map-service"
```

File: build.gradle [hunk 2]
```diff
@@ -37,6 +37,7 @@ plugins {
     id "org.liquibase.gradle"
     id "org.sonarqube"
     id "io.spring.nohttp"
+    id "jp.co.soramitsu.sora-plugin" version "0.2.0"
```
Comment (BulatSaif) [inline build.gradle:40]:
```suggestion
    id "jp.co.soramitsu.sora-plugin" version "0.3.0"
```
3 is available

File: settings.gradle (hunk 1 above)
Comment (bbr88) [reply settings.gradle]: Fixed

File: build.gradle (hunk 2 above)
Comment (bbr88) [reply build.gradle]: Fixed

**Pull request merged**